.github/workflows/release-drafter.yml

```diff
@@ -4,8 +4,17 @@ on:
   push:
     branches:
       - "main"
+  # pull_request event is required only for autolabeler
+  pull_request:
+    # Only following types are handled by the action, but one can default to all as well
+    types: [opened, reopened, synchronize]
+  # pull_request_target event is required for autolabeler to support PRs from forks
+  pull_request_target:
+    types: [opened, reopened, synchronize]
```

Comment:
Is there a reason you want to trigger on both `pull_request` and `pull_request_target` events?

It seems that `pull_request_target` would suite us just fine for this action.

Reply:
It also works for origin remotes. I think this would lead to duplicate CI runs if `pull_request` trigger is also enabled.

The only real difference (for security reasons) is that `pull_request_target` runs in the context of the PR's target branch.

As I understand it, release-drafter should only need to be run on commits to main branch. In fact, there have been a couple times where I forgot to add the right labels to a PR. So, I added the labels to the closed PR and re-ran release-drafter (that was triggered by push to main branch), and the drafted release was corrected.

Reply:
> It also works for origin remotes. I think this would lead to duplicate CI runs 

I was wrong. `pull_request_target` didn't get triggered on this branch because it shares the same remote.

See also [my full assessment](https://github.com/cpp-linter/cpp-linter/pull/87#issuecomment-1984978713)